compressed_string.py

```diff
@@ -0,0 +1,36 @@
+def compress_string(string: str) -> str:
+    """
+    Function which takes a string and then compresses it.
+    """
+    
+    # intialise vairables
+    count = 1
+    compressed_string = ""
+    
+    letters = [char for char in string if char.isalpha()]  # remove all digits from the string
+    letters = "".join(letters)
+
+    for char in range(len(letters)):
```

Comment:
Have you tried using Python's list comprehensions? They are super useful when you want to capture the results of a loop. 

```python
>>> inputs = [1,2,3,4,5]
>>> [str(x) for x in inputs]
['1', '2', '3', '4', '5']
```

More info in this [Python patterns](https://python-3-patterns-idioms-test.readthedocs.io/en/latest/Comprehensions.html) book.